packages/millicast-sdk/integration-tests/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Question - does this need its own tsconfig? Won't it just use the one used by `packages/millicast-sdk` ?

@gregdolby @Santiago-Souto @vishalkharge if you guys know?

Reply:
It looks like it is reccomended for playwright. lhttps://playwright.dev/docs/test-typescript#tsconfigjson